workflows/edit_data_packages/add_a_pre-generated_identifier.Rmd

```diff
@@ -0,0 +1,13 @@
+
+## Add a pre generated identifier to the eml
+
+When you pre generate a UUID or DOI, the change is not automatically reflected in the "Identifier" section of the eml so this makes sure that the eml lines up with the identifier being used. 
+
+```{r, eval = FALSE}
+## Generate DOI and add to EML
+doiPid <- dataone::generateIdentifier(mn, "DOI")
+eml@packageId <- new("xml_attribute",
```

Comment:
Unclear why this is needed. Doesn't publish_update do this automatically.

Reply:
I just encountered a similar problem in my parent EML, and running `publish_update` fixed it.  This workaround might not be required (but it's worth confirming/checking into)